jetty-io/src/main/java/org/eclipse/jetty/io/ArrayRetainableByteBufferPool.java

```diff
@@ -390,13 +392,14 @@ private void evict(boolean direct, long excess)
 
                 if (oldestEntry.remove())
```

Comment:
No null check here?
This same code in `jetty-12.0.x` has a null check. (yes, i'm aware this PR is for `jetty-10.0.x`)

https://github.com/eclipse/jetty.project/blob/e71fa6dedd862aeea05f2ef54dc5789dddef7971/jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java#L393-L397

Reply:
This PR changes `ArrayRetainableByteBufferPool.java` in `jetty-10.0.x` and is missing that null check.

Reply:
I'm going to retract and get more sleep
Disregard everything I've said in the last hour. :-(

Reply:
maybe as a blob:

https://github.com/eclipse/jetty.project/blob/7a7d69a69f4f51772e20813332291189a24e91b1/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayRetainableByteBufferPool.java#L387-L389